resources/home/dnanexus/utils/defaults.py

```
@@ -0,0 +1,69 @@
+        {
+            "name": "Refseq Genes",
+            "format": "refgene",
+            "id": "hg19_genes",
+            "url": "https://s3.amazonaws.com/igv.org.genomes/hg19/ncbiRefSeq.sorted.txt.gz",
+            "indexURL": "https://s3.amazonaws.com/igv.org.genomes/hg19/ncbiRefSeq.sorted.txt.gz.tbi",
+            "visibilityWindow": -1,
+            "supportsWholeGenome": False,
+            "removable": False,
+            "infoURL": "https://www.ncbi.nlm.nih.gov/gene/?term=$$",
+            "type": "annotation",
+        }
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)